addons/token-exchange/blue_secret_controller.go

```diff
@@ -25,6 +27,8 @@ import (
 const (
 	RookType                     = "kubernetes.io/rook"
 	DefaultBlueSecretMatchString = "cluster-peer-token"
+	RookConfigMapName            = "rook-ceph-operator-config"
```

Comment:
This should be defined in mirror peer controller where this configmap is used.

Reply:
done